Core/Sources/Core/RSScreen.swift

```diff
@@ -19,7 +19,7 @@ public class RSScreen {
 import UIKit
 
 public class RSScreen {
-	public static var maxScreenScale = CGFloat(3)
+	public static let maxScreenScale = CGFloat(3)
```

Comment:
Let’s delete RSScreen entirely, and make `maxScreenScale` a static Int property of RSImage-Extensions, since that’s the one place it’s used.